src/ontology/mondo-ingest.Makefile

```diff
@@ -751,6 +752,16 @@ $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.tsv:
 $(EXTERNAL_CONTENT_DIR)/mondo-medgen.robot.tsv:
 	wget "https://github.com/monarch-initiative/medgen/releases/latest/download/medgen-xrefs.robot.template.tsv" -O $@
 
+###### MalaCards #########
```

Comment:
Cool. Just looking at this preliminarily. See if you can re-request or tag me again to review when it is undrafted and ready.

I like their [website](https://www.malacards.org/) and how the entry pages are put together.